.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "submodules/ixdat-large-test-files"]
 	path = submodules/ixdat-large-test-files
-	url = https://github.com/ixdat/ixdat-large-test-files.git
+	url = https://github.com/ixdat/ixdat-large-test-files
```

Comment:
I was experimenting with adding a submodule url with `.git`suffix and without it, and it worked both ways. However, the url was consistent with the one in the `.git/config` file. Are you sure this will not cause problems?